src/msw/webview_ie.cpp

```diff
@@ -1546,10 +1546,18 @@ HRESULT wxSTDCALL DocHostUIHandler::ShowContextMenu(DWORD dwID, POINT *ppt,
     wxUnusedVar(ppt);
     wxUnusedVar(pcmdtReserved);
     wxUnusedVar(pdispReserved);
-    if(m_browser->IsContextMenuEnabled()) 
-        return E_NOTIMPL; 
-    else 
-        return S_OK; 
+    if(!m_browser->IsContextMenuEnabled())
+    {
+        wxContextMenuEvent event(wxEVT_CONTEXT_MENU,
+                             m_browser->GetId(),
+                             m_browser->ClientToScreen(wxDefaultPosition));
+        event.SetEventObject(m_browser);
+        m_browser->HandleWindowEvent(event);
+
+        return S_FALSE;
+    }
+    else
+        return S_OK;
```

Comment:
This contradicts what [the documentation](https://msdn.microsoft.com/en-us/library/aa741313.aspx#Controlling_the_Cont) says:

> Returning E_NOTIMPL or S_FALSE from this method indicates to the WebBrowser Control that it should display its own standard context menu. However, returning S_OK causes the WebBrowser Control not to display its menus, and it assumes that the host has performed the appropriate action.

So, at least according to the docs, the old code, returning `E_NOTIMPL` is the menu is enabled and `S_OK` when it's disabled, was correct and the new one is not as it should still return `E_NOTIMPL` here and `S_OK` instead of `S_FALSE` above.

I don't know if you've actually tested and found out that the real behaviour was different from the documented. If it really was, we should discuss and test this further, but at the very least there should be a comment explaining what's going on here. But I suspect it's really just a mistake that ought to be fixed instead.

Reply:
Sorry for not responsing. Somehow I forgot about this PR and there was no notification until I could not make it work in Mac then I found this same PR thru google. I will work on the comment and rebase to the latest master